### ISSUE TITLE
Webulous Submission for EFO diseases from UniProtKB - Part 2

### ISSUE THREAD

**Issue by ggeorghiou** [EBISPOT/efo#289]
Hello,

Please find the links below for the OMIM terms I've mapped to MONDO for UniProt disease keywords.

https://docs.google.com/spreadsheets/d/1cDkcKwT9fMfDe-imNSXoLoKI7zmZXCle3Ub3ETjm97o/edit?usp=sharing

https://docs.google.com/spreadsheets/d/1r4EGve4G7_YaY91wZWwvlsh5cjMMNC3yPYVP15LJG8s/edit?usp=sharing

https://docs.google.com/spreadsheets/d/1-waeQGeg2yuqVlxPPzsvqzixOIakYZzG76SoO7XdnrA/edit?usp=sharing

Thanks in advance!

Best,

George Georghiou

**Comment by paolaroncaglia** [comment]
@ggeorghiou 
Hi and thanks for sharing those spreadsheets. I'm afraid we won't be able to address them during our current sprint and in time for our upcoming release, but I think we could aim for the mid-December release. I'd suggest we have a chat over Skype or similar, after the mid-November release, so you may explain the details of those spreadsheets to me. Thanks.

**Comment by ggeorghiou** [comment]
Thanks for letting me know Paola! Yes, lets talk at some point then after your November release.

**Comment by paolaroncaglia** [comment]
Update: George will probably meet some EFO team members on Thu Nov 22nd to discuss details of what's needed for this ticket.

**Comment by paolaroncaglia** [comment]
@ggeorghiou 
We were wondering if you have any updates on this please? Unless the priority on it has shifted. We have now released EFO3, so you can look up terms in EFO3 by using the public OLS interface. Any question let us know - and if it's not urgent we can wait of course ;-)
Thanks,
Paola and Zoe

**Comment by ggeorghiou** [comment]
Still working on it, should hope to have something to submit soon. We're busy having an argument internally over what is a disease and what is a phenotype.

**Comment by paolaroncaglia** [comment]
@ggeorghiou welcome to our world :-D If we can be of any help please contact us offline.

**Comment by ggeorghiou** [comment]
Hi all, time to revive this thread - we've reviewed these and have compiled them into one list. I have another list of terms I am going through for a future submission in September. It will be another list of terms that will hopefully not be as long as this one.

Please find the mappings we'd like incorporated to EFO in the google sheet: 

https://drive.google.com/file/d/12hXTQoZdcZV3MkhvpVfQEvcsLJ2yvPns/view?usp=sharing

If there are any concerns, questions or comments, please email me directly since I will be on annual leave for the rest of the month starting the 12th.

**Comment by paolaroncaglia** [comment]
@ggeorghiou 
@zoependlington 
FYI (I know you are both on leave)

I moved this ticket to the "Next" queue. However, I don't have access to the Google sheet above (I just requested it). Given the holidays etc, we'll address this request at some point after George is back. Thanks.

**Comment by paolaroncaglia** [comment]
@ggeorghiou 
Welcome back and thanks for giving access to the Google sheet. I had a very quick look at it. Could you please confirm that what is needed from EFO for that list is to look at only the entries that have a comment in column E starting with "New term request"?
I see that most of those are highlighted in blue, but there's at least two (rows 27 and 40) that aren't. We can sort/search for "New term request" ourselves of course, if you confirm that's what's needed.
Thanks!

**Comment by ggeorghiou** [comment]
Hi Paola,

Those terms highlighted in blue as well as rows 27 and 40 are new term requests. 

The rest are OMIM IDs that I have mapped to ORDO or MONDO terms in EFO that the OMIM ID was not present for when I searched for it in OLS.

**Comment by paolaroncaglia** [comment]
Note for self:
Refer to table https://docs.google.com/spreadsheets/d/12hXTQoZdcZV3MkhvpVfQEvcsLJ2yvPns/edit#gid=1369237308, tab "Paola's copy - new term requests only". It's sorted by disease label alphabetically, to help highlight similar classes.

**Comment by paolaroncaglia** [comment]
@ggeorghiou 
I have a question on one of your requests please:

Otitis media 
"An inflammation of the middle ear resulting in earache, fever, hearing disturbance, and vertigo. "
OMIM:166760
Mapped to exact term 
MONDO:0005441
EFO:0004992

Actually, the OMIM term OMIM:166760 is 'OTITIS MEDIA, SUSCEPTIBILITY TO; OMS', so a quality/trait, while the EFO and MONDO terms listed in this row are for diseases. EFO has MONDO:0008162 'otitis media, susceptibility to' in the 'inherited disease susceptibility' branch that has the OMIM:166760 xref. However, the label and definition provided are for the disease. Could you please confirm if you are looking for the disease susceptibility term or for the disease one?

Thanks,
Paola

**Comment by ggeorghiou** [comment]
Hi Paola, this is specifically for the disease term.

**Comment by paolaroncaglia** [comment]
Thanks @ggeorghiou . The disease term EFO:0004992 'Otitis media' has mappings to Mondo and other resources, but none to OMIM. I will leave the EFO term as is then, unless you indicate if you'd like to add a mapping to an OMIM term different from OMIM:166760. Thanks!

**Comment by paolaroncaglia** [comment]
Quick update:
I spoke to @ggeorghiou , and illustrated the status of this request. 

3 new EFO terms have been added and will be visible after tomorrow's EFO release:
EFO_0010166 Warburg-Cinotti syndrome
EFO_0010167 squalene synthase deficiency
EFO_0010168 spondyloepiphyseal dysplasia, Kondo-Fu type

3 OMIM mappings have been added to existing terms in EFO:
OMIM:618272
OMIM:618150
OMIM:618398

15 Mondo terms have been imported in EFO (complete with OMIM mappings): 
for details, see https://docs.google.com/spreadsheets/d/12hXTQoZdcZV3MkhvpVfQEvcsLJ2yvPns/edit#gid=1521923497

34 OMIM mappings are already in EFO;
for details, see https://docs.google.com/spreadsheets/d/12hXTQoZdcZV3MkhvpVfQEvcsLJ2yvPns/edit#gid=1521923497

Other cases need to be checked. These include 1) OMIM mappings that were evaluated to be exact but where an existing term in EFO has different OMIM mappings, 2) Mondo terms that are only supported by OMIM mappings (need to verify if they represent disease variants and therefore warrant new entries in EFO, or if they are OMIM "numberings" that should map to a broader term), 3) broad OMIM mappings. 
Also, more new terms need to be added. 
We'll aim at getting the second part of this request in the November EFO release. Thanks.

**Comment by paolaroncaglia** [comment]
Note for self:
Currently working on 'mapped to broader term' set, double-checking mappings (a few are already in EFO) and noting action items; down to line 74 in https://docs.google.com/spreadsheets/d/12hXTQoZdcZV3MkhvpVfQEvcsLJ2yvPns/edit#gid=1372175755. TBC.

**Comment by paolaroncaglia** [comment]
Note for self:
Re. broad OMIM mappings for UniProt/Open Targets, I agreed with @zoependlington that when the full list of mappings that need adding is ready, 
- [ ] I'll make a simplified version of the spreadsheet (with OMIM IDs and Orphanet/EFO/Mondo terms only) that she can use to make a ROBOT template.
- [ ] For the subset of broad OMIM mappings that need adding to Mondo terms already in EFO, we may want to ask Mondo if they'd be interested in adding those broad mappings "at the source" in Mondo itself.

**Comment by paolaroncaglia** [comment]
Note for self:
Currently working on completing 'mapped to exact term' set, double-checking action items in view of work done for broad mappings, https://docs.google.com/spreadsheets/d/12hXTQoZdcZV3MkhvpVfQEvcsLJ2yvPns/edit#gid=1521923497, rows 21 to 39 included.

**Comment by paolaroncaglia** [comment]
Update:
- [ ] The draft spreadsheet for the ROBOT template (https://docs.google.com/spreadsheets/d/1D3_DDsXcJ408H54c6ljXfGTMxYYs3bwBxGcPcaqZOa4/edit#gid=0) contains all relevant items from UniProt's "exact" and "broad" mappings.
- [ ] For the "exact" mappings, the only other edits left to do are in tab https://docs.google.com/spreadsheets/d/12hXTQoZdcZV3MkhvpVfQEvcsLJ2yvPns/edit#gid=1521923497, rows 10, 11 and 12.

**Comment by paolaroncaglia** [comment]
@ggeorghiou 
This is now all done. Here's a summary:
The original request contained 331 OMIM IDs that needed mapping, complete with suggestions; 
I reviewed all of them, and in some cases I suggested different mappings (or found that some mappings were already in place); in total, 
23 new EFO terms were created (complete with OMIM mappings);
209 OMIM IDs were mapped to existing terms in EFO (mostly broad mappings);
18 Mondo terms and 1 Orphanet term were imported in EFO (complete with OMIM mappings);
the remaining OMIM IDs were already in EFO at the time of editing (mostly via Mondo terms that had been imported recently).
All of the above will be visible after our upcoming release, scheduled for Monday 18/11.
Thanks to @zoependlington for her help with the ROBOT template to add the revised mappings in EFO.
